src/visions/utils/cache.py

```diff
@@ -51,7 +51,7 @@ def mutable_pseudo_hash(data, node, graph):
     # return id((data, node, graph))
     try:
         if isinstance(data, pd.DataFrame):
-            data_hash = hash(hash(tuple(df[col])) for col in df.columns)
+            data_hash = hash(hash(tuple(data[col])) for col in data.columns)
```

Comment:
Thanks for catching, this was scratch work though, is it being invoked anywhere?